src/main/java/com/iliev/exercises/EvenlyDivisibleNumbers.java

```diff
@@ -0,0 +1,57 @@
+package com.iliev.exercises;
+public class EvenlyDivisibleNumbers {
+    public EvenlyDivisibleNumbers() {
+    }
+
+    public static int[] CalculateNumber(){
+
+      boolean smallestFound = false;
+      boolean biggestFound = false;
+        int[] minAndMaxNumbers = new int[2];
+
+        minNuber(smallestFound, minAndMaxNumbers);
```

Comment:
Instead of passing minAndMaxNumbers you can make the method return the number when it is found.